traffic_portal/test/end_to_end/deliveryServices/pageData.js

```diff
@@ -37,6 +39,10 @@ module.exports = function(){
 	this.protocol=element(by.name('protocol'));
 	this.longDesc=element(by.name('longDesc'));
 	this.remapText=element(by.name('remapText'));
+	this.host=element(by.name('host'));
```

Comment:
can you put these in their own section at the bottom or something? with a comment over them along the lines of:

`// delivery service static dns entry fields`

otherwise, they look like ds properties which they are not.

Reply:
Code updated.